plutus-ir/src/Language/PlutusIR.hs

```diff
@@ -186,8 +189,8 @@ instance TermLike (Term tyname name uni) tyname name uni where
     unwrap   = Unwrap
     iWrap    = IWrap
     error    = Error
-    termLet x (Def vd bind) = Let x NonRec [TermBind x Strict vd bind]
-    typeLet x (Def vd bind) = Let x NonRec [TypeBind x vd bind]
+    termLet x (Def vd bind) = Let x NonRec (pure $ TermBind x Strict vd bind)
```

Comment:
Hmm, ordinarily I prefer to use functions like `singleton` for containers when we're not thinking of them as "non-determinism effects", but there doesn't seem to be one for `NonEmpty`. Slightly horrifyingly, it does have an `IsList` instance that just throws error on an empty list, so you *could* use `OverloadedLists` here, but I'm not sure that's better.